### DISCUSSION TITLE
Adding crypto currencies as supported payment method of sponsorship

### DISCUSSION THREAD

**Discussion by hayyaun** [community/community#3813] [Other Feature Feedback, Questions, & Ideas]
Hi, I wonder if there was a method where we could use crypto assets.

**Comment by bdresser** [comment]
@hayyaun we don't have any near-term plans to support crypto payments, but it's on our radar :) 

Are you asking because you'd like to Sponsor someone else using crypto, or because you want to accept sponsorships others pay in crypto?

**Reply by hayyaun** [reply]
Both of them tbh. I wanted to be sponsor that I found it impossible.  If I want sponsor maybe in the future  the crypto seems to be the only way.

**Reply by sr229** [reply]
This needs to be a feature as an alternative to waiting for fintech providers like Stripe to cover territories, which was the problem I outlined in #3719 when GitHub considered Stripe as the only payment processor.

**Reply by hayyaun** [reply]
> This needs to be a feature as an alternative to waiting for fintech providers like Stripe to cover territories, which was the problem I outlined in #3719 when GitHub considered Stripe as the only payment processor.

This will be a good movement, but there are still countries blocked by Swift program and have no access to Paypal, using crypto-only fintech can be a better solution, it makes it possible for almost anyone to use it.

**Comment by tallguyjenks** [comment]
Thus far the only crypto option I've seen/used is using BAT with the brave browser through GitHub. Some people have sent me BAT through GitHub specifically

![image](https://user-images.githubusercontent.com/29872822/123503720-85a27b80-d609-11eb-970a-5a9511db8dde.png)

**Reply by tallguyjenks** [reply]
$0 because i cash out on the regular 🤣

**Comment by Poussinou** [comment]
@bdresser

Hi, [F-Droid](https://f-droid.org/) contributor here. [We are using](https://f-droid.org/docs/Inclusion_Policy/) the FUNDING.yml file hosted directly in the GitHub repos to show the donation information for each app on our website. We support more ways to donate than the 10 ways currently officially available in the FUNDING.yml file format.

The question is : Can we add new entries in the FUNDING.yml files to support paypal, bitcoin or something else without breaking something else on your side?

**Comment by jfullstackdev** [comment]
also I want to add my idea to let GitHub Sponsors non-GitHub users to sponsor us, so, we are not limited to GitHub users sponsoring us, like, just simply entering details when paying for an online purchase through Debit or Credit Card, even PayPal has that feature, even without PayPal account, a donor can donate with ease 

https://github.com/community/community/discussions/27603
